src/types/models/applications.ts

```diff
@@ -27,8 +29,15 @@ interface AccessRule {
   container: string;
   transform: TransformType;
   permissions: string[];
+  conditions: Condition[];
 }
 
+type Condition = {
+  attribute: string;
+  operator: string;
+  value: string;
+};
+
 type CreateApplication = Pick<Application, 'name' | 'type'> &
```

Comment:
```suggestion
type CreateApplication = Required<Pick<Application, 'name' | 'type'>> &
```

This would make `name` and `type` mandatory when creating an application

Reply:
`name` shouldn't be required